packages/react-native-editor/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ For each user feature we should also add a importance categorization label  to i
 -->
 
 ## Unreleased
+
+## 1.67.0
 -   [*] [Embed block] Included Link in Block Settings [#36099]
 -   [**] Fix tab titles translation of inserter menu [#36534]
 -   [*] [Media&Text block] Fix an issue where the text font size would be bigger than expected in some cases [#36570]
```

Comment:
@SiobhyB what do you think about updating this changelog entry to add the spaces? LMK 🙇🏾 
```suggestion
-   [*] [Media & Text block] Fix an issue where the text font size would be bigger than expected in some cases [#36570]
```

Reply:
Sounds good @jd-alexander! Updated in https://github.com/WordPress/gutenberg/pull/36848/commits/93a041a4173e7747267cb8279451753df263c88b and also added a missing change in https://github.com/WordPress/gutenberg/pull/36848/commits/88cb6a632b8d4a1505a25c63637358f6de4a7916. I'll go through to update the connected PRs too.